app/controllers/events_controller.rb

```diff
@@ -28,6 +28,22 @@ def create
     end
   end
 
+  def edit
+  end
+
+  def update
+    if @event.update(event_params)
+      redirect_to @events, notice: "Event was successfully updated."
+    else
+      render 'edit'
```

Comment:
Please use double quotes throughout. 😀 This was also caught by RuboCop, so make sure to always rune `bundle exec rubocop`. 👮